src/features/certification/certification.tsx

```diff
@@ -6,17 +6,22 @@ import styles from './certification.module.scss';
 
 interface RequiredProps {
   courseName: CourseNames;
-  lang?: 'ru' | 'en';
 }
 
 const localizedContent = {
-  en: {
-    title: 'Communication',
+  default: {
+    title: 'Certification',
     firstParagraph:
       "To earn a course certificate, you must complete all assignments, finish the final project, and achieve at least 70% of the top student's score in the course. The certificate is a recognition of your hard work and dedication.",
     secondParagraph: '',
   },
-  ru: {
+  'js / front-end ru': {
+    title: 'Сертификат',
+    firstParagraph:
+      'Чтобы получить сертификат о прохождении курса вам необходимо набрать 70% от результата TOP-1 студента. Например, если в конце этапа у лучшего студента 2000 баллов, проходной для всех студентов 1400 баллов (2000 * 0.7).',
+    secondParagraph: '',
+  },
+  'js / front-end pre-school ru': {
     title: 'Сертификат',
     firstParagraph:
       'Чтобы получить сертификат о прохождении подготовительного этапа вам необходимо набрать 70% от результата TOP-1 студента. Например, если в конце этапа у лучшего студента 2000 баллов, проходной для всех студентов 1400 баллов (2000 * 0.7).',
```

Comment:
Can we be sure, that Pre-school evaluation has so high score?

Reply:
I'd rather include a translation of the English version, and delete these calculations)